_docs/introduction_to_rest_apis/docapis_intro_to_rest_api_doc.md

```diff
@@ -62,7 +62,7 @@ Each REST API functions a bit differently. There isn't a single way of doing thi
 
 Another reason why REST APIs are taking off is because the web itself is evolving into a conglomeration of APIs. Instead of massive, do-it-all systems, web sites are pulling in the services they need through APIs.
 
-For example, rather than building your own search to power your website, you might use Switftype instead and leverage their service through the [Swiftype API](https://swiftype.com/developers). Rather than building your own payment gateway, you might integrate [Stripe and its API](https://stripe.com/docs/api). Rather than building your own login system, you might use [UserApp and its API](https://app.userapp.io/#/docs/). Rather than building your own e-commerce system, you might use [Snipcart and its API](http://docs.snipcart.com/api-reference/introduction). And so on.
```

Comment:
The link https://swiftype.com/developers now redirects to a different one. I'm not sure if the page is the same, please check.

Reply:
Fixed 'Switftype' → 'Swiftype' here.

Reply:
Userapp is discontinued and the link is no more. Maybe, replace it with [Auth0](https://auth0.com/docs/api-auth)?